### ISSUE TITLE
Implement PFFT ?

### ISSUE THREAD

**Issue by veintemillas** [veintemillas/jaxions#19]
See https://github.com/mpip/pfft

**Comment by AlexVaq** [comment]
I don't think this is necessary.

**Comment by AlexVaq** [comment]
Upping this because of the problem with the axion spectral propagator

**Comment by AlexVaq** [comment]
It seems that there isn't an optimal solution for MPI and FFT. The PFFT library is a slight improvement over FFTW, but one needs an underlying FFT library to make it work (in other words, we need to install FFTW anyway). The difference in performance is visible once we use more than 256 MPI processes, and although we might reach that number in the future, our problem main is memory, not performance.

In other words, it's a small improvement that can consume a lot of time and make the setup in a particular machine more difficult. I wouldn't spend time on this now. We can keep an eye on parallel FFT solutions for the future.

Check https://www-user.tu-chemnitz.de/~potts/workgroup/pippig/paper/PFFT_SIAM_88588.pdf for a performance comparison between FFTW and PFFT.
